planet-clicker/Assets/_Script/Data/CountData.cs

```diff
@@ -5,22 +5,21 @@ namespace _Script.Data
 {
     public class CountData : DataModel
     {
-        // NOTE: Explicitly set to lower case for compatibility.
-        public long count { get; private set; }
+        public long Count { get; private set; }
 
         public CountData(long c)
```

Comment:
As backward compatibility isn't much of a concern for this toy project, we can probably change this to `long count`.

Reply:
yes good